examples/images_contours_and_fields/quiver_demo.py

```diff
@@ -35,7 +35,7 @@
                pivot='mid', units='inches')
 qk = ax2.quiverkey(Q, 0.9, 0.9, 1, r'$1 \frac{m}{s}$', labelpos='E',
                    coordinates='figure')
-ax2.scatter(X[::3, ::3], Y[::3, ::3], color='r', s=5)
+ax2.plot(X[::3, ::3], Y[::3, ::3], 'r.', ms=3)
```

Comment:
I'd prefer to keep the explicit kwargs (ie. `color='r', marker='.'`) as then it's obvious what is happening - it's definitely not intuitive what `'r.'` means.

Reply:
switched to `".", c="r"` (unfortunately `marker="."` is not sufficient as one also need to pass `linestyle="none"`, which becomes quite a mouthful).